include/nbl/asset/IMeshPacker.h

```diff
@@ -78,8 +32,23 @@ class MeshPackerBase
     };
 
 protected:
-    MeshPackerBase(const AllocationParams& allocParams)
-        :m_allocParams(allocParams) {};
+    MeshPackerBase(uint16_t minTriangleCountPerMDIData, uint16_t maxTriangleCountPerMDIData)
+        :m_maxTriangleCountPerMDIData(maxTriangleCountPerMDIData),
+         m_minTriangleCountPerMDIData(minTriangleCountPerMDIData),
+         m_MDIDataAlctrResSpc(nullptr),
+         m_idxBuffAlctrResSpc(nullptr),
+         m_vtxBuffAlctrResSpc(nullptr)
```

Comment:
in V2 you can allocate vertex and index data from the same buffer (it will actually save you one descriptor set binding because you won't need to have two UTBs for `R32_UINT` or `R16_UINT` if user wants to use UTBs instead of SSBOs)

Reply:
ignore what I said... I forgot that we'll still be using the hardware vertex input for the indices 🤦